esp_delta_ota/CHANGELOG.md

```diff
@@ -3,3 +3,10 @@
 ### Enhancements:
 - Added support to pass user data to write callback with new callback function: `merged_stream_write_cb_with_user_ctx_t`
 - The older write callback function: `merged_stream_write_cb_t` has been deprecated and will be removed in the next major release.
+
+## 1.1.1
+
+### Enhancements:
```

Comment:
Since you are creating a bugfix update here, we can change this to `## Bugfixes`